### PR TITLE
Added seekto command

### PR DESCRIPTION
**Pull request by smogenheimer**
### This pull request...
adds a command: "seekto" to seek to a specific position in the currently song.

### CODE REVIEW

**Comment by CyberFlameGO** [comment]
Doesn't #674 have this as W.I.P already anyways?

**Comment by lokha** [comment]
The code does not compile

```
[ERROR] /F:/IdeaProjects/MusicBot-seek/src/main/java/com/jagrosh/jmusicbot/JMusicBot.java:[119,43] ')' expected
[ERROR] /F:/IdeaProjects/MusicBot-seek/src/main/java/com/jagrosh/jmusicbot/JMusicBot.java:[120,43] ';' expected
```

File: src/main/java/com/jagrosh/jmusicbot/JMusicBot.java [hunk 1]
```diff
@@ -116,6 +116,7 @@ public static void main(String[] args)
                         new PauseCmd(bot),
                         new PlaynextCmd(bot),
                         new RepeatCmd(bot),
+                        new SeekToCmd(bot);
```
Comment (Sanduhr32) [inline src/main/java/com/jagrosh/jmusicbot/JMusicBot.java:119]: Needs to be a `,`, not a semi colon `;`.

File: src/main/java/com/jagrosh/jmusicbot/commands/dj/SeekToCmd.java [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+    @Override
+    public void doCommand(CommandEvent event) 
+    {
+        String index = "";
+        try
+        {
+            index = event.getArgs();
+            if(index.indexOf(':') == -1){
+                throw new NumberFormatException("Wrong format. Example: 5:18 (up to hours only).");
```
Comment (Sanduhr32) [inline src/main/java/com/jagrosh/jmusicbot/commands/dj/SeekToCmd.java:52]: Should not fail, arithmetically divide the number into hours, minutes, etc.

If you wanted it to fail, use IllegalArgumentException.
NumberFormatException is for stuff like "0b7f" on binary, "3F" on dec,  or "0xGGCZ" on hex.

**Review by Sanduhr32 — CHANGES_REQUESTED** [review]
:/

**Comment by MichailiK** [comment]
#674 already exists, which is mostly done & polished.

**Comment by MichailiK** [comment]
Since my PR #674 is (mostly) finished, I'll close this in favor of it.

**Pull request closed without merging**